introduction.md

```diff
@@ -3,3 +3,10 @@ My favorite Code School path is https://www.codeschool.com/paths/ruby
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?